altium.py

```diff
@@ -259,6 +259,7 @@ def get_int_frac(obj, property):
     '''Return full value of a field with separate integer and fraction'''
     value = obj.get_int(property)
     value += obj.get_int(property + "_FRAC") / FRAC_DENOM
+    value += obj.get_int(property + "_FRAC1") / FRAC_DENOM #sometimes FRAC1 is used instead of FRAC. 
```

Comment:
Your code below has `get_int_frac(obj, "DISTANCEFROMTOP")*10`. Suppose there is DISTANCEFROMTOP=10 and DISTANCEFROMTOP_FRAC1=500000 (500 thousand). According to format.md, this should give a final result of 105 in the usual units. But according to my reading of the code, you are returning value = 10 + 500,000/100,000 = 15, and the caller will multiply that by ten to get 150. Something’s not right.